### PR TITLE
auth faq: document a ProtectSystem=full implication

### PR DESCRIPTION
**Pull request by Habbie**
### Checklist

I have:
- [ ] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: docs/appendices/FAQ.rst [hunk 1]
```diff
@@ -72,6 +72,13 @@ Linux Netfilter says your conntrack table is full?
 Thats a common problem with Netfilter Conntracking and DNS Servers, just tune your kernel variable (``/etc/sysctl.conf``) ``net.ipv4.netfilter.ip_conntrack_max`` up accordingly.
 Try setting it for a million if you don't mind spending some MB of RAM on it for example.
 
+I get an error about writing to /etc
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This may look something like "unable to open temporary zonefile '/etc/powerdns/zones/example.com.<random number>'".
+Our systemd units enable ``ProtectSystem=full`` by default, which disallows writes to ``/etc`` and ``/usr``, among other places.
```
Comment (jsoref) [inline docs/appendices/FAQ.rst:79]:
```suggestion
PowerDNS systemd units enable ``ProtectSystem=full`` by default, which disallows writes to ``/etc`` and ``/usr``, among other places.
```

**Review by jsoref — COMMENTED** [review]
General 👍

File: docs/appendices/FAQ.rst [hunk 2]
```diff
@@ -72,6 +72,13 @@ Linux Netfilter says your conntrack table is full?
 Thats a common problem with Netfilter Conntracking and DNS Servers, just tune your kernel variable (``/etc/sysctl.conf``) ``net.ipv4.netfilter.ip_conntrack_max`` up accordingly.
 Try setting it for a million if you don't mind spending some MB of RAM on it for example.
 
+I get an error about writing to /etc
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This may look something like "unable to open temporary zonefile '/etc/powerdns/zones/example.com.<random number>'".
+PowerDNS systemd units enable ``ProtectSystem=full`` by default, which disallows writes to ``/etc`` and ``/usr``, among other places.
+Either move your zone files to a safer place (``/var/lib/powerdns`` is a popular choice) or change the systemd protection settings.
```
Comment (pieterlexis) [inline docs/appendices/FAQ.rst:80]: link to https://www.freedesktop.org/software/systemd/man/systemd.exec.html#ProtectSystem= and mentioning `ReadWriteDirectories` might help as well

Comment (Habbie) [reply docs/appendices/FAQ.rst:80]: done

**Pull request merged**